apps/site/src/app/(main)/volunteer/VolunteerForm.tsx

```diff
@@ -0,0 +1,20 @@
+import BasicInformation from "../apply/sections/Form/BasicInformation";
+import AgeInformation from "../apply/sections/Form/AgeInformation";
+import SchoolInformation from "../apply/sections/Form/SchoolInformation";
+import ShiftAvailability from "./components/ShiftAvailability";
+import VolunteerFRQ from "./components/VolunteerFRQ";
+import ExtraQuestions from "../apply/sections/Form/ExtraQuestions";
+import Form from "../apply/sections/Form/Form";
```

Comment:
Chore: it would make more sense for the components for any form sections used by multiple application types to be placed in a common directory elsewhere rather than being part of the Apply page.

Reply:
Refactored common forms into a dedicated forms folder